core/server/data/meta/context_object.js

```diff
@@ -1,8 +1,16 @@
-var config = require('../../config'),
-    _      = require('lodash');
+var settingsCache = require('../../api/settings').cache,
+    _ = require('lodash');
 
 function getContextObject(data, context) {
-    var blog = config.get('theme'),
+    /**
+     * @TODO:
+     *   - the only valid property is cover
+     *   - there was no image property in config.get('theme')
+     *   - there was no author property in config.get('theme')
+     */
```

Comment:
I have added facebook and twitter 👍 
But title and logo is not used?

What i did: i looked for `getContextObject` usages in the code base. 
It would be great if you could double check :)